tls/s2n_kem.c

```diff
@@ -74,23 +162,56 @@ const struct s2n_kem_group s2n_secp256r1_kyber_512_r3 = {
     .kem = &s2n_kyber_512_r3,
 };
 
-#if EVP_APIS_SUPPORTED
+const struct s2n_kem_group s2n_secp256r1_kyber_768_r3 = {
+    .name = "secp768r1_kyber-768-r3",
+    .iana_id = TLS_PQ_KEM_GROUP_ID_SECP256R1_KYBER_768_R3,
+    .curve = &s2n_ecc_curve_secp256r1,
+    .kem = &s2n_kyber_768_r3,
+};
+
+const struct s2n_kem_group s2n_secp384r1_kyber_768_r3 = {
+    .name = "secp384r1_kyber-768-r3",
+    .iana_id = TLS_PQ_KEM_GROUP_ID_SECP384R1_KYBER_768_R3,
+    .curve = &s2n_ecc_curve_secp384r1,
+    .kem = &s2n_kyber_768_r3,
+};
+
+const struct s2n_kem_group s2n_secp521r1_kyber_1024_r3 = {
+    .name = "secp521r1_kyber-1024-r3",
+    .iana_id = TLS_PQ_KEM_GROUP_ID_SECP521R1_KYBER_1024_R3,
+    .curve = &s2n_ecc_curve_secp521r1,
+    .kem = &s2n_kyber_1024_r3,
+};
+
 const struct s2n_kem_group s2n_x25519_kyber_512_r3 = {
     .name = "x25519_kyber-512-r3",
     .iana_id = TLS_PQ_KEM_GROUP_ID_X25519_KYBER_512_R3,
     .curve = &s2n_ecc_curve_x25519,
     .kem = &s2n_kyber_512_r3,
 };
-#else
-const struct s2n_kem_group s2n_x25519_kyber_512_r3 = { 0 };
-#endif
+
+const struct s2n_kem_group s2n_x25519_kyber_768_r3 = {
+    .name = "x25519_kyber-768-r3",
```

Comment:
Same comment. Did we want to change the name to `X25519Kyber768Draft00`? If we're using [the ID from the draft RFC, should we also use their name too](https://datatracker.ietf.org/doc/html/draft-tls-westerbaan-xyber768d00-02#section-5-2.2.1)?